src/entity/Teacher.java

```diff
@@ -0,0 +1,28 @@
+package entity;
+
+import java.util.List;
+
+public class Teacher extends entity.Employee {
+
+    private List<entity.Courses> courses; // cursuri pe care le preda
+    private List<entity.Schedule> orar;
+
+    public Teacher(String name, String departament, int salary, List<entity.Courses> courses) {
+        super(name, departament, salary);
+        this.courses = courses;
+        this.orar = orar;
+    }
+
+    public List<entity.Courses> getCourses() { return courses;}
+    public List<entity.Schedule> getOrar() { return orar;}
```

Comment:
no need to write entity.Schedule/Course, doar Course respectiv Schedule (te rog sa redenumesti si Courses in Course)